editor/state.js

```diff
@@ -148,6 +148,15 @@ export function mode( state = 'visual', action ) {
 	return state;
 }
 
+export function sidebar( state = { opened: false }, action ) {
```

Comment:
Let's call it PostInspector when the time comes. The idea being that the entire sidebar is the inspector, but when nothing is selected, you are inspecting the post itself, therefore showing the meta boxes.